eng/Version.Details.xml

```diff
@@ -22,20 +22,20 @@
       <Sha>c01a538851a8ab1a1fbeb2e6243f391fff7587b4</Sha>
       <RepoName>aspnet-xdt</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="5.0.0-rtm.21221.2" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+    <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="5.0.0-rtm.21310.1" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
       <Uri>https://github.com/mono/linker</Uri>
-      <Sha>6a15056a9d71169bf04d007301edfd558afcfa6a</Sha>
+      <Sha>e6a47c70b7878b346ff0858085eb8a6379f6f0bb</Sha>
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.7" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
-      <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>556582d964cc21b82a88d7154e915076f6f9008e</Sha>
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.8" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
```

Comment:
These shouldn't reference the internal repos should they?

Reply:
Fixed, thanks.